brainstormforce/ultimate-addons-for-gutenberg
# Fix: Rendered the Inspector Tabs at Priority in the Inspector Controls.

### Description
Rendered the Inspector Tabs at Priority in the Inspector Controls.

### Types of changes
Bugfix (non-breaking change which fixes an issue)

### How has this been tested?
 - Activate Kadence/Toolset plugin.
 - Switch to UAG next-release.
 - Take a new Page & Drag-drop info-fox block.
 - You will see all the Editors Kit / Toolset plugin settings about out Tabs - https://share.bsf.io/BludJDPQ
 - Try the same in this branch, the issue should be fixed. 

### Checklist:
